### PR TITLE
Setup Release.yml workflow

### PR DESCRIPTION
**Pull request by Egorand**
Borrowed from https://github.com/cashapp/sqldelight/blob/master/.github/workflows/Release.yml

### CODE REVIEW

File: .github/workflows/Release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+      - name: Publish the artifacts
+        env:
+          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: ${{ secrets.ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME }}
+          ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: ${{ secrets.ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD }}
```
Comment (JakeWharton) [inline .github/workflows/Release.yml:19]: Did you initialize these in the repo?

File: .github/workflows/Release.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          # TODO(egorand): Figure out if the publishing plugin needs GPG keys and how they get fed
```
Comment (JakeWharton) [inline .github/workflows/Release.yml:20]: You cannot publish without the artifacts being signed

File: .github/workflows/Release.yml (hunk 1 above)
Comment (Egorand) [reply .github/workflows/Release.yml:19]: Alec did

File: .github/workflows/Release.yml (hunk 2 above)
Comment (Egorand) [reply .github/workflows/Release.yml:20]: Yeah makes sense. I'll add the keys.

**Comment by JakeWharton** [comment]
Obsolete now, right?

**Comment by Egorand** [comment]
Yes, closing

**Pull request closed without merging**